lib/ROCrates.Net.Tests/ROCrates.Net.Tests.csproj

```diff
@@ -35,6 +35,8 @@
     <None Update="Fixtures/test-person.json" CopyToOutputDirectory="PreserveNewest"/>
     <None Update="Fixtures/test-computer-language.json" CopyToOutputDirectory="PreserveNewest"/>
     <None Update="Fixtures/test-computational-workflow.json" CopyToOutputDirectory="PreserveNewest"/>
+    <None Update="Fixtures/test-root-dataset.json" CopyToOutputDirectory="PreserveNewest"/>
```

Comment:
If you expecting more of these fixture build items, might be worth doing it just as a wildcard `/*.json` at some point.

Reply:
It worked!